classes/Cart.php

```diff
@@ -332,6 +332,36 @@ public function updateAddressId($id_address, $id_address_new)
         Db::getInstance()->execute($sql);
     }
 
+    /**
+     * Update the Delivery Address ID of the Cart.
+     *
+     * @param int $id_address Current Address ID to change
+     * @param int $id_address_new New Address ID
+     */
+    public function updateDeliveryAddressId($id_address, $id_address_new)
```

Comment:
```suggestion
    public function updateDeliveryAddressId(int $id_address, int $id_address_new)
```

Reply:
I just copied the function above but alright 😉